### PR TITLE
scide: adding solarizedLight and solarizedDark themes

### PR DESCRIPTION
**Pull request by redFrik**
i add it here but we can close directly if the dracula theme is preferred. https://github.com/supercollider/supercollider/pull/3410

### CODE REVIEW

**Comment by gusano** [comment]
@redFrik we should also add both solarized ones as well :)

**Comment by brianlheim** [comment]
I would support adding both :)

**Comment by redFrik** [comment]
i should've mentioned that this is after https://github.com/helmholtz/sc-solarized

**Comment by brianlheim** [comment]
Same as with #3410, just want to make sure that licensing has been properly considered.

**Comment by patrickdupuis** [comment]
As was discussed on slack, adding this and other themes via separate C++ files requires a little more work than expected. It was decided to simply add the license text above the function that contains the theme's colour palette.

I've created a PR against @redFrik's branch which adds the license text.

**Comment by patrickdupuis** [comment]
Ping @redFrik. We'd like to get this into 3.9.1 if possible.

**Comment by redFrik** [comment]
thanks for taking care of adding the licence.  i'm happy to see this go into 3.9.1.  is there something i need to do though?  like some button click on github?

**Comment by patrickdupuis** [comment]
First, you need to merge redFrik/supercollider#2 so that those changes become part of this PR. There may be some sort of conflict that needs to be resolved before this can be merged into `3.9`.

**Comment by redFrik** [comment]
ok @patrickdupuis merged and resolved. hope it's good now. thanks for your help

**Comment by patrickdupuis** [comment]
@snappizz Good to go?

File: editors/sc-ide/core/settings/theme.cpp [hunk 1]
```diff
@@ -271,6 +355,11 @@ Theme::Theme(const QString & _name, Manager * settings)
         mLocked = true;
     } else if (mName == "dracula") {
         fillDracula();
+    } else if (mName == "solarizedLight") {
```
Comment (brianlheim) [inline editors/sc-ide/core/settings/theme.cpp]: There needs to be a line `mLocked = true` in the block above.

**Pull request merged**